inc/composer/class-docker-compose-generator.php

```diff
@@ -480,7 +480,7 @@ protected function get_service_s3() : array {
 	protected function get_service_tachyon() : array {
 		return [
 			'tachyon' => [
-				'image' => 'humanmade/tachyon:2.3.2',
+				'image' => 'humanmade/tachyon:2.4.0',
```

Comment:
@roborourke this docker image does not seem to exist.

Reply:
🤦‍♂️  I see we have a bit of mix of release tags prepended with `v` on this repo but without on Docker hub. Should have been `v2.4.0`

Reply:
```suggestion
				'image' => 'humanmade/tachyon:v2.4.0',
```

Reply:
I'm still using the `edge` builds, haven't had a chance to try the versions Rob tagged, but I see 2.4.0 [here](https://hub.docker.com/layers/humanmade/tachyon/v2.4.0/images/sha256-e43878b7dc1a6ae8cb8b17a80ac28fff9ec133f63b4ef8be5f50dfd8f55b0bcd?context=explore)